src/10-serial-communication/nix-tooling.md

```diff
@@ -1,5 +1,25 @@
 # *nix tooling
 
+## Newer revisions of the discovery board
+
+With newer revisions, if you connect the discovery board to your laptop / PC you
+should see a new TTY device appear in `/dev`.
+
+``` console
+$ # Linux
+$ dmesg | tail | grep -i tty
+[13560.675310] cdc_acm 1-1.1:1.2: ttyACM0: USB ACM device
+```
+
+This is the USB <-> Serial device. On Linux, it's named `ttyACM*`.
```

Comment:
Is it always ttyACM on linux? I know some devices/distros use `ttyUSB` instead, though I am not aware of what causes this distinction.

Reply:
Device names are defined by the driver. If the device driver is the generic CDC ACM driver then the devices will typically be called `ttyACMx`, other drivers may use other conventions (I would guess mostly to prevent naming conflicts), however there's another layer:  Devices are (usually) managed by `udev` which can also override the device names assigned by the driver, so if a distribution wants to make naming uniform they can actually do that.

Reply:
TIL, thanks @therealprof  :)